sigle/src/hooks/users.ts

```diff
@@ -25,3 +25,29 @@ export const useGetUserByAddress = (
     () => UserService.getApiUsers({ userAddress: stacksAddress }),
     options
   );
+
+type GetApiUsersFollowingReturnType = Awaited<
+  ReturnType<typeof UserService.getApiUsersFollowing>
+>;
+export const useGetUsersFollowing = (
```

Comment:
I was following the methods attached to `UserService` for naming the queries and noticed this one is very similar to the `useGetUserFollowing` in the appData file - would it best to use a different naming convention in this case or do you think it's fine?

Reply:
Good point, maybe we should use a different naming for the Gaia ones, eg: `useGaiaGetUserFollowing` or smth like this